### DISCUSSION TITLE
实现DAG图

### DISCUSSION THREAD

**Discussion by Jzhenli** [didi/LogicFlow#1414] [General]
大家好，初次接触，试问logicflow能否做出类似下面的DAG效果图
![image](https://github.com/didi/LogicFlow/assets/42105688/1ba8a78e-36fc-43e5-af57-5bf40328a6ac)

网页链接参考：https://xflow.antv.vision/~demos/dag-layout

**Comment by boyongjiong** [comment]
可以的，可以看下示例demo或者文档，或者进用户群问一下有没有类似的实现，可以交流
